docs/pbc/all/payment-service-provider/202404.0/base-shop/third-party-integrations/payone/integration-in-the-back-office/integrate-payone.md

```diff
@@ -2,53 +2,37 @@
 title: Integrate Payone
 description: Learn how you can integrate the Payone app into your Spryker shop
 template: howto-guide-template
-last_updated: Jan 09, 2024
+last_updated: Aug 27, 2024
 redirect_from:
   - /docs/pbc/all/payment-service-providers/payone/integrate-payone.html
   - /docs/pbc/all/payment-service-provider/202311.0/third-party-integrations/payone/integration-in-the-back-office/integrate-payone.html
 ---
 
-This document describes how to integrate the Payone app into a Spryker shop.
+This document describes how to integrate Payone using the Payone app.
 
 ## Prerequisites
-Before integrating Algolia, ensure the following prerequisites are met:
-- Make sure your project is ACP-enabled. See [App Composition Platform installation](/docs/acp/user/app-composition-platform-installation.html) for details.
 
-- The Payone app requires the following Spryker modules:
+- Fulfill [App Composition Platform prerequisites](/docs/dg/dev/acp/sccos-dependencies-required-for-the-acp.html).
+- Install the modules for Payone. To check the list of required modules and their versions, in the Back Office, go to **Apps**>**Payone**. The list of modules is displayed in **Requirements** > **Spryker module list** section.
+- Optional: To show payment details in the Back Office Order Details page, install the `spryker/sales-payment-detail: ^1.2.0` module.
```

Comment:
Please add one more step at the bottom of the document for this optional requirement to integrate it using [this guide](https://docs.spryker.com/docs/pbc/all/payment-service-provider/202404.0/base-shop/retrieve-and-use-payment-details-from-third-party-psps.html)